app/components/search-result.js

```diff
@@ -27,6 +27,16 @@ export default Ember.Component.extend(Analytics, {
         return result.description.slice();
     }),
 
+    shortContributorList: Ember.computed('result', function() {
+        let filtered = this.set('justContributors', this.get('result.contributors').filter(item => !!item.users.bibliographic));
+        let nameArr = this.set('shortContributors', filtered.slice(0, Math.min(10, filtered.length)));
```

Comment:
What is the advantage of having both `shortContributorsList` and `shortContributors`? They seem to be storing the same variable?